.travis.yml

```diff
@@ -21,16 +21,22 @@ env:
     - TEST_SUITE=integration INTEGRATION_INDEX=2
     - TEST_SUITE=integration INTEGRATION_INDEX=3
     - TEST_SUITE=static
+    - TEST_SUITE=js
 cache:
   apt: true
-  directories: $HOME/.composer/cache
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.nvm
```

Comment:
I forgot about cashing `node_modules` and maybe also `yarn.lock` (it's like `composer.lock` for Yarn, but works better 😜 ).